*: update FAQ of tools (#4453)
pingcap/docs: tidb-lightning/tidb-lightning-faq.md

2. Delete the entire "import" directory on the machine hosting `tikv-importer`.

3. Delete all tables and databases created on the TiDB cluster, if needed.

## Why does TiDB Lightning report the `could not find first pair, this shouldn't happen` error?

> The reason for the error is that an anomaly occurs when scanning the locally sorted files, possibly because the number of files opened in Lightning exceeds the system limit. In Linux system, you can use the `ulimit -n` command to confirm whether this value is too small. It is recommended that you adjust this value to `1000000` (`ulimit -n 1000000`) during Lightning import.

Reviewer: @glorv PTAL
```suggestion
This error occurs possibly because the number of files opened by TiDB Lightning exceeds the system limit when going through the sorted local files. In the Linux system, you can use the `ulimit -n` command to confirm whether the value of this system limit is too small. It is recommended that you adjust this value to `1000000` (`ulimit -n 1000000`) during Lightning import.
```

Reviewer 2: LGTM

Reviewer: Thanks for reminding. Changed to `read`.

> This error occurs possibly because the number of files opened by TiDB Lightning exceeds the system limit when TiDB Lightning goes through the sorted local files. In the Linux system, you can use the `ulimit -n` command to confirm whether the value of this system limit is too small. It is recommended that you adjust this value to `1000000` (`ulimit -n 1000000`) during Lightning import.

Reviewer 2: Shall we replace `go through` with `read` or `iterate`? it's a bit weird in this context.

Reviewer 2: Not sure `read`, `scan`, `iterate`, `go through` which is better. Seldom seen `go through` for files..

> This error occurs possibly because the number of files opened by TiDB Lightning exceeds the system limit when TiDB Lightning goes through the sorted local files. In the Linux system, you can use the `ulimit -n` command to confirm whether the value of this system limit is too small. It is recommended that you adjust this value to `1000000` (`ulimit -n 1000000`) during Lightning import.

Reviewer: ```suggestion
This error occurs possibly because the number of files opened by TiDB Lightning exceeds the system limit when TiDB Lightning reads the sorted local files. In the Linux system, you can use the `ulimit -n` command to confirm whether the value of this system limit is too small. It is recommended that you adjust this value to `1000000` (`ulimit -n 1000000`) during Lightning import.
```

## Why does TiDB Lightning report the `could not find first pair, this shouldn't happen` error?

> This error occurs possibly because the number of files opened by TiDB Lightning exceeds the system limit when going through the sorted local files. In the Linux system, you can use the `ulimit -n` command to confirm whether the value of this system limit is too small. It is recommended that you adjust this value to `1000000` (`ulimit -n 1000000`) during Lightning import.

Reviewer: ```suggestion
This error occurs possibly because the number of files opened by TiDB Lightning exceeds the system limit when TiDB Lightning goes through the sorted local files. In the Linux system, you can use the `ulimit -n` command to confirm whether the value of this system limit is too small. It is recommended that you adjust this value to `1000000` (`ulimit -n 1000000`) during Lightning import.
```
